packages/assets/src/generateTs.ts

```diff
@@ -1,6 +1,6 @@
 import fs from 'fs/promises';
 import path from 'path';
-import {AssetType} from './utils';
+import {AssetType, normalizeToUnix} from './utils';
```

Comment:
Why are we importing `normalizeToUnix` and re-exporting it from `utils`? Perhaps it is cleaner to import it from `import normalizeToUnix from 'normalize-path';` directly in this file.

Reply:
In this case it is mainly to make interaction with utils more defined and communicate intention of normalizeToUnix. At the point of introducing it it wasn't strictly clear if using external dependency was needed or not, but used it for convenience. I think doing reexport has minimal cost, but allows for being an adaption layer for easier refactoring if it where to be removed.

Reply:
Alright, good that this is an active choice :)